plugins/runtime/gov.nasa.jpl.imce.oml.runtime/model/OMLRuntime.xcore

```diff
@@ -95,57 +97,12 @@ class OMLAspect extends OMLEntity {}
  */
 class OMLConcept extends OMLEntity {}
 
-abstract class OMLRelationshipFromEntity extends OMLObject {
-	refers OMLEntity[1] omlSource
-}
-
-abstract class OMLRelationshipFromStructure extends OMLObject {
-	refers OMLStructure[1] omlSource
-}
-
-abstract class OMLRelationshipToEntity extends OMLObject {
-	refers OMLEntity[1] omlTarget
-}
-
-abstract class OMLRelationshipToStructure extends OMLObject {
-	refers OMLStructure[1] omlTarget
-}
-
-abstract class OMLEntityRelationship extends OMLRelationshipFromEntity, OMLRelationshipToEntity {}
-
 /*
  * @see gov.nasa.jpl.imce.oml.model.descriptions.ReifiedRelationshipInstance
  * @see gov.nasa.jpl.imce.oml.model.descriptions.ReifiedRelationshipInstanceDomain
  * @see gov.nasa.jpl.imce.oml.model.descriptions.ReifiedRelationshipInstanceRange
  */
-class OMLReifiedRelationship extends OMLEntityRelationship, OMLEntity {}
-
-/*
- * @see gov.nasa.jpl.imce.oml.model.descriptions.UnreifiedRelationshipInstanceTuple
- */
-class OMLUnreifiedRelationship extends OMLEntityRelationship {}
-
-/*
- * @see gov.nasa.jpl.imce.oml.model.descriptions.SingletonInstanceScalarDataPropertyValue
- */
-class OMLEntityDataPropertyToScalar extends OMLRelationshipFromEntity {
-	LexicalValue value
-}
-
-/*
- * @see gov.nasa.jpl.imce.oml.model.descriptions.SingletonInstanceStructuredDataPropertyValue
- */
-class OMLEntityDataPropertyToStructure extends OMLRelationshipFromEntity, OMLRelationshipToStructure {}
-
-
-/*
- * @see gov.nasa.jpl.imce.oml.model.descriptions.ScalarDataPropertyValue
- */
-class OMLStructureDataPropertyToScalar extends OMLRelationshipFromStructure {
-	LexicalValue value
+class OMLReifiedRelationship extends OMLEntity {
```

Comment:
This looses the capability of properly representing the differences in the original oml runtime.